pkg/config/config.go

```diff
@@ -753,33 +754,34 @@ func DefaultConfig() (*Config, error) {
 			ConmonEnv: []string{
 				"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
 			},
-			ConmonCgroup:             "system.slice",
-			SELinux:                  selinuxEnabled(),
-			ApparmorProfile:          apparmor.DefaultProfile,
-			BlockIOConfigFile:        DefaultBlockIOConfigFile,
-			IrqBalanceConfigFile:     DefaultIrqBalanceConfigFile,
-			RdtConfigFile:            rdt.DefaultRdtConfigFile,
-			CgroupManagerName:        cgroupManager.Name(),
-			PidsLimit:                DefaultPidsLimit,
-			ContainerExitsDir:        containerExitsDir,
-			ContainerAttachSocketDir: conmonconfig.ContainerAttachSocketDir,
-			MinimumMappableUID:       -1,
-			MinimumMappableGID:       -1,
-			LogSizeMax:               DefaultLogSizeMax,
-			CtrStopTimeout:           defaultCtrStopTimeout,
-			DefaultCapabilities:      capabilities.Default(),
-			LogLevel:                 "info",
-			HooksDir:                 []string{hooks.DefaultDir},
-			NamespacesDir:            defaultNamespacesDir,
-			DropInfraCtr:             true,
-			seccompConfig:            seccomp.New(),
-			apparmorConfig:           apparmor.New(),
-			blockioConfig:            blockio.New(),
-			rdtConfig:                rdt.New(),
-			ulimitsConfig:            ulimits.New(),
-			cgroupManager:            cgroupManager,
-			deviceConfig:             device.New(),
-			namespaceManager:         nsmgr.New(defaultNamespacesDir, ""),
+			ConmonCgroup:               "system.slice",
+			SELinux:                    selinuxEnabled(),
+			ApparmorProfile:            apparmor.DefaultProfile,
+			BlockIOConfigFile:          DefaultBlockIOConfigFile,
+			IrqBalanceConfigFile:       DefaultIrqBalanceConfigFile,
+			RdtConfigFile:              rdt.DefaultRdtConfigFile,
+			CgroupManagerName:          cgroupManager.Name(),
+			PidsLimit:                  DefaultPidsLimit,
+			ContainerExitsDir:          containerExitsDir,
+			ContainerAttachSocketDir:   conmonconfig.ContainerAttachSocketDir,
+			MinimumMappableUID:         -1,
+			MinimumMappableGID:         -1,
+			LogSizeMax:                 DefaultLogSizeMax,
+			CtrStopTimeout:             defaultCtrStopTimeout,
+			DefaultCapabilities:        capabilities.Default(),
+			LogLevel:                   "info",
+			HooksDir:                   []string{hooks.DefaultDir},
+			NamespacesDir:              defaultNamespacesDir,
+			DropInfraCtr:               true,
+			SeccompUseDefaultWhenEmpty: seccompConfig.UseDefaultWhenEmpty(),
+			seccompConfig:              seccompConfig,
+			apparmorConfig:             apparmor.New(),
+			blockioConfig:              blockio.New(),
+			rdtConfig:                  rdt.New(),
+			ulimitsConfig:              ulimits.New(),
+			cgroupManager:              cgroupManager,
+			deviceConfig:               device.New(),
+			namespaceManager:           nsmgr.New(defaultNamespacesDir, ""),
```

Comment:
It would be nice to have these in alphanumeric order.  A change like this would be easier to spot, plus it makes my OCD happy.

Reply:
Fixed :+1: